files-community/Files
# Feature: Added support for opening Git root in VS Code

**Resolved / Related Issues**
   Closes #13102...

**Validation**
How did you test these changes?
   1. Open sub folder in a Git epo
   2. Click the open button on the status bar
   3. Confirm that choosing "open repo" opened the root of the Git repo
   4. Confirm that choosing "open folder" opens the current directory

**Screenshots (optional)**
Add screenshots here.
